CommonTools/RecoAlgos/python/HBHENoiseFilterResultProducer_cfi.py

```diff
@@ -6,13 +6,10 @@
     minHPDHits = cms.int32(17),
     minHPDNoOtherHits = cms.int32(10),
     minZeros = cms.int32(9999),
-    IgnoreTS4TS5ifJetInLowBVRegion = cms.bool(True),
-    defaultDecision = cms.string("HBHENoiseFilterResultRun1"),
+    IgnoreTS4TS5ifJetInLowBVRegion = cms.bool(False),
+    defaultDecision = cms.string("HBHENoiseFilterResultRun2Loose"),
     minNumIsolatedNoiseChannels = cms.int32(10),
     minIsolatedNoiseSumE = cms.double(50.0),
-    minIsolatedNoiseSumEt = cms.double(25.0)
+    minIsolatedNoiseSumEt = cms.double(25.0),
+    useBunchSpacingProducer = cms.bool(True)
 )
-
-from Configuration.StandardSequences.Eras import eras
-eras.run2_common.toModify(HBHENoiseFilterResultProducer, IgnoreTS4TS5ifJetInLowBVRegion=False)
```

Comment:
I'm still expecting the 
``` eras.run2_common.toModify(HBHENoiseFilterResultProducer, IgnoreTS4TS5ifJetInLowBVRegion=False) ``` 
to return back to the file and "True" to be in the run1 config.